CoThings/Views/Settings/ServerSettingsView.swift

```diff
@@ -27,26 +30,65 @@ struct ServerSettingsView: View {
     }
     
 	var body: some View {
-        Form {
-            Section(header: Text("Server URL")) {
-                HStack(alignment: .firstTextBaseline, spacing: 1) {
-                    Text("https://")
-                    TextField("demo-eu.cothings.app", text: $serverHostname)
-                        .keyboardType(.URL)
-                        .disableAutocorrection(true)
-                        .autocapitalization(.none)
-                        .frame(maxWidth: .infinity)
-                }
+        ZStack{
+            NavigationLink("Scan QR Code", destination: QRCodeView(
```

Comment:
What do you think about presenting Scan QR Code screen as a modal?

Reply:
Oh yes this will be much better! In order to do this I added a title and a dismiss button in the controller : 
![99289016_180122119973758_9107629862297796608_n](https://user-images.githubusercontent.com/44622425/82761968-6cc00900-9dfe-11ea-880d-11387f1c1139.jpg)

Reply:
This looks great!